.github/workflows/auto-updater.yml

```diff
@@ -0,0 +1,47 @@
+name: Automatic Data Update
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  DataUpdater:
+    name: Automatic Data Update
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'sapphiredev'
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+      - name: Use Node.js v20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: yarn
+          registry-url: https://registry.yarnpkg.com/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Run plugin-api MIME type sync
+        working-directory: ./packages/api
+        run: yarn sync-mime-types
```

Comment:
Can also be turned into `yarn workspace @sappphire/api sync-mime-types` fyi

Reply:
I knew of the command and I wanted to use it, but I wasn't remembering it, thanks!